keras_cv/models/backbones/mobilenet_v3/mobilenet_v3_backbone_test.py

```diff
@@ -0,0 +1,137 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import tensorflow as tf
+from absl.testing import parameterized
+from tensorflow import keras
+
+from keras_cv.models.backbones.mobilenet_v3.mobilenet_v3_backbone import (
+    MobileNetV3Backbone,
+)
+from keras_cv.models.backbones.mobilenet_v3.mobilenet_v3_backbone import (
+    MobileNetV3LargeBackbone,
+)
+from keras_cv.models.backbones.mobilenet_v3.mobilenet_v3_backbone import (
+    MobileNetV3SmallBackbone,
+)
+from keras_cv.utils.train import get_feature_extractor
+
+
+class MobileNetV3BackboneTest(tf.test.TestCase, parameterized.TestCase):
+    def setUp(self):
+        self.input_batch = tf.ones(shape=(2, 224, 224, 3))
+
+    def test_valid_call(self):
+        model = MobileNetV3Backbone(
+            stack_fn=None,
+            last_point_ch=1024,
+            include_rescaling=False,
+        )
+        model(self.input_batch)
+
+    def test_valid_call_with_rescaling(self):
+        model = MobileNetV3Backbone(
+            stack_fn=None,
+            last_point_ch=1024,
+            include_rescaling=True,
+        )
+        model(self.input_batch)
+
+    @parameterized.named_parameters(
+        ("tf_format", "tf", "model"),
+        ("keras_format", "keras_v3", "model.keras"),
+    )
+    def test_saved_model(self, save_format, filename):
+        model = MobileNetV3Backbone(
+            stack_fn=None,
+            last_point_ch=1024,
+            include_rescaling=True,
+        )
+        model_output = model(self.input_batch)
+        save_path = os.path.join(self.get_temp_dir(), filename)
+        model.save(save_path, save_format=save_format)
+        restored_model = keras.models.load_model(save_path)
+
+        # Check we got the real object back.
+        self.assertIsInstance(restored_model, MobileNetV3Backbone)
+
+        # Check that output matches.
+        restored_output = restored_model(self.input_batch)
+        self.assertAllClose(model_output, restored_output)
+
+    @parameterized.named_parameters(
+        ("tf_format", "tf", "model"),
+        ("keras_format", "keras_v3", "model.keras"),
+    )
+    def test_model_backbone_layer_names_stability(self):
+        model = MobileNetV3Backbone(
+            stack_fn=None,
+            last_point_ch=1024,
+            include_rescaling=False,
+        )
+        model_2 = MobileNetV3Backbone(
+            stack_fn=None,
+            last_point_ch=1024,
+            include_rescaling=False,
+        )
+        layers_1 = model.layers
+        layers_2 = model_2.layers
+        for i in range(len(layers_1)):
+            if "input" in layers_1[i].name:
+                continue
+            self.assertEquals(layers_1[i].name, layers_2[i].name)
+
+    def test_create_backbone_model_with_level_config(self):
```

Comment:
I expect this test should fail after /gcbrun due to the lack of `model.pyramid_level_outputs`, but the test case is a good way to understand the expected behavior of this part of the backbone.